iotics/lib/identity/register/document_helper.py

```diff
@@ -80,9 +134,29 @@ def get_valid_issuer_key_for(doc: RegisterDocument, issuer_name: str, get_contro
     @staticmethod
     def get_valid_issuer_key_for_control_only(doc: RegisterDocument, issuer_name: str,
                                               get_controller_doc: GetControllerDocFunc) -> Optional[IssuerKey]:
+        """
+        Get an IssuerKey matching issuer name (control only)
+        :param doc: existing register document
+        :param issuer_name: name of issuer
+        :param get_controller_doc: resolver discover function
+        :return: IssuerKey or None
+
+        :raises:
+            IdentityValidationError: if invalid name or did
+        """
         return RegisterDocumentHelper.get_valid_issuer_key_for(doc, issuer_name, get_controller_doc, include_auth=False)
 
     @staticmethod
     def get_valid_issuer_key_for_auth(doc: RegisterDocument, issuer_name: str,
                                       get_controller_doc: GetControllerDocFunc) -> Optional[IssuerKey]:
+        """
+        Get an IssuerKey matching issuer name (control and authentication)
```

Comment:
`Get a valid issuer key matching issuer name from the control and authentication keys and delegation proofs`